### PR TITLE
fix serialization issues

### PR DESCRIPTION
**Pull request by sterlp**
- https://github.com/bekk/db-scheduler-ui/issues/105
- https://github.com/bekk/db-scheduler-ui/issues/107
- fixed how the static resources are copied, using maven tools to ensure it works on any OS.

### CODE REVIEW

File: db-scheduler-ui-frontend/package.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "postbuild": "mkdir -p dist/db-scheduler-ui && mv dist/* dist/db-scheduler-ui/ 2>/dev/null || true",
```
Comment (geirsagberg) [inline db-scheduler-ui-frontend/package.json:9]: Why is this removed?

Comment (geirsagberg) [reply db-scheduler-ui-frontend/package.json:9]: ah, related to making the copying of static tools work on any OS I assume

Comment (sterlp) [reply db-scheduler-ui-frontend/package.json:9]: Yes, this has not worked on windows...
